src/backend/templates/tracks.hbs

```diff
@@ -118,6 +118,7 @@
 		             	              {{#speakers_list}}
 		             	                {{name}}
 		             	                {{#if organisation}}({{organisation}}){{/if}}
+                                          {{#if @last}}{{else}},{{/if}}
```

Comment:
@aayusharora  space between first speaker and comma ?
like this https://rishabhk07.github.io/open-gh/tracks.html